### PR TITLE
Add test data for register userkeys

### PR DESCRIPTION
**Pull request by Pearl1594**

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@Pearl1594 is this solving #96 or does it relate in some other way?

**Comment by Pearl1594** [comment]
I only raised this PR to run the tests.. Will close it once I receive some update on the issue #96

**Comment by DaanHoogland** [comment]
@Pearl1594 , we shouldn´t add the test dat for future testing either?

**Comment by Pearl1594** [comment]
We could add it.

File: test/testdata/UserService.json [hunk 1]
```diff
@@ -149,5 +149,13 @@
...
+  },
+  "registerUserKeys": {
+    "registeruserkeysresponse": {
+      "userkeys": {
+        "apikey": "hNfJi2dO5kwD6gTtYoifqu_4CSI3FcTtxc4MydW24PdJalv8mfTGsf425twAdLrIKX5IewfqIfYDEumO6-yRaw",
+        "secretkey": "zV8kAYlyY4H-rEDO-Ot-VVPjKteVcU4haQcdRarte9wuOUhko3uzcnnFqNii8xifcpbe0eWR6IHAXjVGB2m1Hw"
+      }
+    } 
   }
 }
```
Comment (DaanHoogland) [inline test/testdata/UserService.json:161]: just n case we start adding pre-comit hooks here as well,... ;)
```suggestion
}

```

**Review by DaanHoogland — COMMENTED** [review]
clgtm, (de)pending possible addition of tests.

**Review by DaanHoogland — APPROVED** [review]

**Pull request merged**